src/main/kotlin/kweb/client/Server2ClientMessage.kt

```diff
@@ -22,11 +24,21 @@ import kotlinx.serialization.json.JsonElement
 @SerialName("Server2ClientMessage")
 data class Server2ClientMessage(
         val yourId: String,
+        val functionCalls : List<FunctionCall>
+) {
+    constructor(yourId : String, functionCall : FunctionCall) : this(yourId, listOf(functionCall))
```

Comment:
Is this constructor override necessary given the next one? Seems like the vararg should handle the case of a single FunctionCall.

Reply:
I wrote both to have options. Currently, I'm not using the vararg constructor. The way we handle Server2ClientMessage creation, we really only need the secondary constructor that takes one item.

The only place in the code base where we generate a `Server2ClientMessage` with more than one `FunctionCall` is in `render()`, and I make a list manually there, so I use the default constructor.

So I was thinking of deleting the vararg constructor. You're right that we could use only the vararg constructor though.

Reply:
I'm ok with either option.